rebalance/rebalance.py

```diff
@@ -666,6 +666,46 @@ def rebalancestop(plugin: Plugin):
     return {"message": plugin.rebalanceall_msg}
 
 
+@plugin.method("rebalancereport")
+def rebalancereport(plugin: Plugin):
+    """Show information about rebalance
+    """
+    res = {}
+    res["rebalanceall_is_running"] = plugin.mutex.locked()
+    res["getroute_method"] = plugin.getroute.__name__
+    res["maxhops_threshold"] = plugin.maxhops
+    res["msatfactor_threshold"] = plugin.msatfactor
```

Comment:
this is not a threshold value ;)  We can name it just `msatfactor` or `capacity_factor`. In the latter case we should also rename it in the plugin remaining code. I think its a better speaking name.

Reply:
I started to write `threshold` everywhere, because I thought the `erringnodes` expression is misleading here. :)

Reply:
Yes, the names aren't optimal